pkg/addon/addon.go

```diff
@@ -109,6 +109,8 @@ const (
 	ObservabilityAddonEndpointComponent = "grafana"
 	// ObservabilityAddonDomainArg is the domain argument name of the observability addon
 	ObservabilityAddonDomainArg = "domain"
+	// LocalAddonRegistryName is the addon-registry name for those installed by local dir
+	LocalAddonRegistryName = "local"
```

Comment:
What if a user already used `local` as his/her registry name?

Reply:
Cannot got the reason why user name an addon-registry with `local`.